### ISSUE TITLE
Can't build on Windows

### ISSUE THREAD

**Issue by pwain** [styled-components/polished#107]
Had some trouble building polished on Windows 10. Several problems exist...

**rm and cp commands not available on Windows:**
I get the following error on initial install, and subsequent builds:

```
> polished@ prebuild:lib C:\_Dev\OSS\polished
> rm -rf lib/*

'rm' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Peter\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-c
li.js" "run" "build:lib"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE
npm ERR! polished@ prebuild:lib: `rm -rf lib/*`
npm ERR! Exit status 1
```

The cp command used during `postbuild:docs:site` has a similar problem.

Both can be worked around by using [shx](https://github.com/shelljs/shx) for cross-platform command support within the npm scripts.

**documentationjs does not support glob paths on Windows**
If I work my way through the issues above, I'm presented with this:

```
> polished@ build:docs:site C:\_Dev\OSS\polished
> documentation build src/** -t docs-theme --github -o docs -f html -c ./.documentation.json

C:\_Dev\OSS\polished\node_modules\documentation\lib\input\expand_directories.js:35
      throw new Error('Input file ' + index + ' not found!');
      ^

Error: Input file src/** not found!
    at C:\_Dev\OSS\polished\node_modules\documentation\lib\input\expand_directories.js:35:13
    at arrayMap (C:\_Dev\OSS\polished\node_modules\lodash\lodash.js:660:23)
    at map (C:\_Dev\OSS\polished\node_modules\lodash\lodash.js:9556:14)
    at Function.flatMap (C:\_Dev\OSS\polished\node_modules\lodash\lodash.js:9259:26)
    at expandDirectories (C:\_Dev\OSS\polished\node_modules\documentation\lib\input\expand_directories.js:17:1
2)
    at dependencyStream (C:\_Dev\OSS\polished\node_modules\documentation\lib\input\dependency.js:54:3)
    at expandInputs (C:\_Dev\OSS\polished\node_modules\documentation\index.js:56:3)
    at build (C:\_Dev\OSS\polished\node_modules\documentation\index.js:107:10)
    at Object.build (C:\_Dev\OSS\polished\node_modules\documentation\lib\commands\build.js:89:3)
    at Object.<anonymous> (C:\_Dev\OSS\polished\node_modules\documentation\bin\documentation.js:13:29)

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Peter\\AppData\\Roaming\\npm\\node_modules\\np
m\\bin\\npm-cli.js" "run" "build:docs:site"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE
npm ERR! polished@ build:docs:site: `documentation build src/** -t docs-theme --github -o docs -f html -c ./.d
ocumentation.json`
npm ERR! Exit status 1
```

This issue is addressed in the latest build of documentationjs ([v4.0.0-beta.18](https://github.com/documentationjs/documentation/releases/tag/v4.0.0-beta.18))

I've made changes to fix these problems on my Windows box, and have confirmed that they also work on OSX. I'll submit a PR shortly so you can take a closer look.

**Comment by pwain** [comment]
PR: #108

**Comment by bhough** [comment]
Merged #108

**Comment by nikgraf** [comment]
awesome!
